src/developer/web-api/new-tracker.md

```diff
@@ -336,7 +336,78 @@ Tracker notes do not have their dedicated endpoint; they are exchanged as part o
 
 > One between `uid` or `username` field must be provided. If both are provided, only username is considered.
 
+### Program stage working lists { #webapi_working_list_filters } 
 
+The program stage working lists feature within the Capture app is designed to display pre-established working lists relevant to a particular program stage in the tracker user interface. This functionality enables users to save standard filters and sorting preferences that are related to program stages, facilitating the organization and management of their workflow. To interact with them, you'll need to use the */api/programStageWorkingLists* resource. These lists can be shared and follow the same sharing pattern as any other metadata object. When using the */api/sharing* the type parameter will be *programStageWorkingLists*.
+
+    /api/40/programStageWorkingLists
+
+##### Payload on CRUD operations to program stage working lists
+
+If you want to fetch or make changes to a program stage working list, besides the endpoint mentioned above, you'll need to provide a payload in the following format: 
+
+
+
+Table: Payload
+
+| Payload values | Description | Example |
+|---|---|---|
+| name | Name of the working list. Required. ||
+| description | A description of the working list. ||
+| program | Object containing the id of the program. Required. | {"id" : "uy2gU8kTjF"} |
+| programStage | Object containing the id of the program stage. Required. | {"id" : "oRySG82BKE6"} |
+| programStageQueryCriteria | An object representing various possible filtering values. See *Program Stage Query Criteria* definition table below.
+
+Table: Program Stage  
```

Comment:
```suggestion
Table: Program Stage Query Criteria
```

Reply:
@enricocolasante I don't think we should expose internal type names. So we can change such names without having to change the docs. I might get this wrong here. We should only expose our HTTP API like query params

Reply:
You mean it for program stage query criteria? It's also used in the API call, so the clients need to be aware of it.